src/clients/SpokePoolClient.ts

```diff
@@ -648,11 +653,18 @@ export class SpokePoolClient extends BaseAbstractClient {
           this.latestDepositIdQueried = deposit.depositId;
         }
       }
+    };
+
+    // Query "legacy" V3FundsDeposited events.
+    for (const event of ["V3FundsDeposited", "FundsDeposited"]) {
```

Comment:
`FundsDeposited` are new right? Comment is confusing me

Reply:
Ah - I'll nuke the comment.

Reply:
https://github.com/across-protocol/sdk/pull/865/commits/c4d6936e07d558291f523f90aa57acd13f7a3e73